src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,93 @@
+import { Outlet } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PersonInfo } from '../PersonInfo/PersonInfo';
+
+export const PeoplePage = () => {
+  const [peopele, setPeople] = useState<Person[] | null>([]);
+  const [loading, setLoading] = useState<boolean>(false);
+  const [errorMessage, setErrorMessage] = useState<string>('');
```

Comment:
you can miss generic type when you add some initial value
```suggestion
  const [loading, setLoading] = useState(false);
  const [errorMessage, setErrorMessage] = useState('');
```